### PR TITLE
Move ingress-exporter to app catalog

### PR DESCRIPTION
**Pull request by wtty-fool**
Towards https://github.com/giantswarm/giantswarm/issues/10177

## TODO:
- [x] migrate to from dep go modules (done in #11)
- [x] :warning: merge https://github.com/giantswarm/ingress-exporter/pull/10
- [x] remove old chart
- [x] **opsctl**: add `ingress-exporter` to app catalog https://github.com/giantswarm/opsctl/pull/652
- [x] **draughtsman/installations**: remove `ingress-exporter` from its list https://github.com/giantswarm/installations/pull/1239
- [x] test on test installation - tested on _geckon_

### CODE REVIEW

**Comment by kopiczko** [comment]
Is this a copy only?

**Comment by wtty-fool** [comment]
It's also a base branch for https://github.com/giantswarm/ingress-exporter/pull/10. This way we get clean diff in #10 and then I can do the rest here.

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,17 +1,53 @@
...
-    - run: ./architect build
-    - deploy:
-        command: |
-          if [ "${CIRCLE_BRANCH}" == "master" ]; then
-            ./architect deploy
-          fi
+      - architect/push-to-app-collection:
+          context: architect
+          name: push-ingress-exporter-to-kvm-app-collection
+          app_name: "ingress-exporter"
```
Comment (kopiczko) [inline .circleci/config.yml:44]: I forgot we need to add this:

```
```suggestion
          app_namespace: "monitoring"
```

Comment (wtty-fool) [reply .circleci/config.yml:44]: Done done

**Pull request merged**